### PR TITLE
users: create strelaysrv manpage

### PR DESCRIPTION
**Pull request by aviau**

### CODE REVIEW

File: users/strelaysrv.rst [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    Per session rate limit, in bytes/s.
+
+.. cmdoption:: -ping-interval=<duration>
+
+    How often pings are sent (default 1m0s).
+
+.. cmdoption:: -pools=<pool addresses>
+
+    Comma separated list of relay pool addresses to join (default "https://relays.syncthing.net/endpoint").
+
```
Comment (calmh) [inline users/strelaysrv.rst]: Blank to disable announcement to a pool, thereby remaining a private relay. Important to note as this is probably the most common use of the option.

**Comment by calmh** [comment]
:+1: merge after the rename

File: users/strelaysrv.rst [hunk 2]
```diff
@@ -77,6 +79,80 @@ Options
...
+although your milage may vary.
+
+Recommended Specifications
+--------------------------
+
+To run a relay of your own, you will first need a server. Running a relay from
+a home computer is  not recommended: the relay ideally needs to be running 24/7,
+and home internet connections often have poor upload bandwidth.
+
+The recommended specifications for a relay are:
```
Comment (calmh) [inline users/strelaysrv.rst]: I think we should remove this section while anyway doing this maneuver. It's questionably accurate and prone to quickly become out of date.

Comment (aviau) [inline users/strelaysrv.rst]: I had put it at the bottom for this very reason, but now that you let me remove it I will :)

File: users/strelaysrv.rst [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+Options
+-------
+
+.. cmdoption:: -debug
+
+    Enable debug output.
+
+.. cmdoption:: -ext-address=<address>
+
+    An optional address to advertising as being available on. Allows listening
```
Comment (phil-davis) [inline users/strelaysrv.rst]: s/to advertising/to advertise
The typo/grammar was there in the previous text as well, and is in the source code.
Source code text fix proposed in https://github.com/syncthing/syncthing/pull/3377

**Comment by calmh** [comment]
@st-review merge it

**Comment by st-review** [comment]
:ok_hand: Merged as a1ac0aa44612392dc99953e4b26ea83d077de123. Thanks, @aviau!

**Pull request closed without merging**